packages/sdk/tests/e2e/dataProtectorCore/grantAccess.test.ts

```diff
@@ -98,6 +98,54 @@ describe('dataProtectorCore.grantAccess()', () => {
     MAX_EXPECTED_WEB2_SERVICES_TIME
   );
 
+  it(
+    'Should be able to authorize specific user when `any` is already authorized',
+    async () => {
+      // grantAccess to any user
+      await dataProtectorCore.grantAccess({
+        ...input,
+        authorizedUser: ethers.ZeroAddress,
+        authorizedApp: sconeAppAddress,
+      });
+      // grantAccess to specific user
+      const grantedAccess = await dataProtectorCore.grantAccess({
+        ...input,
+        authorizedApp: sconeAppAddress,
+      });
+
+      expect(grantedAccess.requesterrestrict).toEqual(
+        input.authorizedUser.toLowerCase()
+      );
+    },
+    2 * MAX_EXPECTED_WEB2_SERVICES_TIME
+  );
+
+  it.only(
```

Comment:
please remove the it.only from the test to ensure all tests are executed.


```suggestion
  it(
```